apps/vapi/RenderManager.cpp

```diff
@@ -18,7 +18,10 @@ using namespace VAPoR;
 
 RenderManager::RenderManager(ControlExec *ce) : _controlExec(ce) {}
 
-RenderManager::~RenderManager() { delete _glManager; }
+RenderManager::~RenderManager()
+{
+    if (_glManager) delete _glManager;
```

Comment:
Doesn't _glManager need to be explicitly initialized by the constructor?

Reply:
The constructor isn't currently guaranteed to have an OpenGL context.

Reply:
Yes, i know. Doesn't it need to be zeroed (set to nulptr) in the constructor?

Reply:
It is initialized to `nullptr` in the class declaration.